Will Heikoff Documentation

```diff
@@ -57,4 +57,51 @@ Replaced “G:\MAX-Filer\Research\Labs-kbuzard-S18\CARR\California\Base Data\Lab
 	by Buzard et al. which sites the use of only 645 labs rather than the 687 labs given in Labs.shp
 Continuing troubleshooting issue with PVAL syntax issue on line 191
 
+6/25/18
+Continued troubleshooting issue with PVAL syntax on line 191 with Prof. Buzard
+Found issue with importing shapefile on line 20
+Resolved issue by using Windows’ Command Prompt to install pyshp, the python shapefile library. Accomplished this by 
+	changing the current directory to the C: drive, by using the function “c:”, then using the 
+	“\Apps-SU\Anaconda3>python –m pip install pyshp” function to install the pyshp library
+		Used “\Apps-SU\Anaconda3” as this is where the pip package is located
+Created new Python Path for shapefile, using the location of where pyshp was installed and stored by the command prompt
+Changed ‘sys.path.append’ location to "G:\MAX-Filer\Research\Labs-kbuzard-S18\kfunctions" in order to create path to import
```

Comment:
This location is going to have to be updated since the folder moved. This line should also be made more precise: this only lets us import the kfunctions (not other modules)